packages/dds/merge-tree/src/client.ts

```diff
@@ -567,6 +567,7 @@ export class Client {
                     "RangeOutOfBounds",
                     {
                         usageError: true,
+                        errorSource: "ddsInvalidOp",
```

Comment:
How should customer think about this source? Is this FF error? Or usage error?
I think we should be more clear by either using FF/ddsInvalidOp, or Usage/ddsInvalidOp (and likely Usage/ddsUser)
(replace "Usage" with something better, maybe "DataStoreUser", not sure)